body.tex

```diff
@@ -156,63 +161,66 @@ \subsubsection{Verification Elements Analysis}
 Verification Elements are baselined in a \textit{Verification Baseline Document}.
 
 \subsubsection{Test Cases Consolidation}
 
-The owner of each test case is in charge to complete it, providing all relevant information. 
+The owner of each test case must complete it, providing all relevant information.
 This includes also drafting the steps in the detailed test procedure.
 Additional test cases can be created, and related to a verification element in order to guarantee the traceability.
 In general, test cases can be related with multiple verification elements, and therefore with more than one requirement.
 
-Test cases shall be general, and just specify the software component or the dataset type required. 
-However, the exact software or dataset version, or the configuration to be used during the test shall never specified a test case. 
-That type of information is provided the test campaign planing phase.
-When test cases are instantiated in a test campaign, small adjustments may be done.
+Test cases should be general, and just specify the software component or the dataset type required.
+The exact software and/or dataset versions and the configuration to be used during the test
+is provided in the test campaign planing phase.
+When test cases are instantiated in a test campaign, small adjustments may be made.
+%clarify what kind of adjustments
 Test Cases are baselined in \textit{Test Specifications}.
 
 \subsubsection{Planning and Execution}
 
 As we already mentioned above, the test activities are organized in test campaigns.
-For each test campaign, two Jira ATM objects have to be created:
+For each test campaign, two Jira ATM objects are created:
 
 \begin{itemize}
 \item \textbf{Jira ATM Test Plan} that provides the context of the test activity, and usually corresponds to a project milestone.
-\item \textbf{Jira ATM Test Cycle}(s) that provides the scope. For each test campaign we may have multiple test cycles, 
-depending on the different configurations, datasets, or extra conditions we may want to test. Each Test Cycle is traced
+\item \textbf{Jira ATM Test Cycle}(s) that provides the scope. For each test campaign we may have multiple test cycles,
+with different configurations, datasets, or extra conditions we may want to test. Each Test Cycle is traced
 to the Jira ATM Test Plan, and provides the list of test cases that need to be executed.
 \end{itemize}
 
 For each test campaign we identify two phases:
 
 \paragraph{Planning}
-It is the phase when the test campaign is prepared. All relevant information, like for example software version, datasets to use, hardware or configuration, 
-is be collected in the Jira ATM Test Plan and Test Cycle(s). 
+It is the phase when the test campaign is prepared. All relevant information, e.g. software version, datasets to use, hardware or configuration,
+is  collected in the Jira ATM Test Plan and Test Cycle(s).
+%not obvious you can collect stuf fin two places ..
```

Comment:
It is because the test campaign is defined using both objects.
I rephrased a little bit.